packages/grafana-ui/src/components/PanelChrome/PanelNotices.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { QueryResultMetaNotice } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+import { Icon } from '../Icon/Icon';
+import { ToolbarButton } from '../ToolbarButton';
+
+interface Props {
+  notices: QueryResultMetaNotice[];
+  onClick?: (e: React.SyntheticEvent, tab: string) => void;
+}
+
+export function PanelNotices({ notices, onClick }: Props) {
+  const styles = useStyles2(getStyles);
+
+  const renderNotice = (notice: QueryResultMetaNotice) => {
+    const iconName =
+      notice.severity === 'error' || notice.severity === 'warning' ? 'exclamation-triangle' : 'info-circle';
+
+    if (notice.inspect && onClick) {
+      return (
+        <ToolbarButton
```

Comment:
Toolbar button changes look based on if topnav is enabled , not sure that good choice for panel header.  

Maybe a regular button with fill text could work, hmmm

Reply:
🤔 I did not know we do that. I will double-check this point.

Reply:
@torkelo I didn't see changes in the `ToolbarButton` when I enabled the `topnav`, am I missing something?

https://user-images.githubusercontent.com/239999/211595397-5df5fa34-2b07-4c5d-a09b-6c807826ce20.mp4

Reply:
@axelavargas strange, the button should have a border when topnav is disabled

Reply:
Oh right! I added a CSS class to remove that border, in that way I was aligned with the design.

Reply:
hm.. maybe stupid, let's keep ToolbarButton for now (except for timeshift, as it is just a info text and not button right now)